spec/services/applications/change_funded_place_spec.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Applications::ChangeFundedPlace do
+  let(:params) do
+    {
+      application:,
+    }
+  end
+
+  subject(:service) do
+    described_class.new(params)
+  end
+
+  describe "#change" do
+    let(:application) do
+      create(:application,
+             :accepted,
+             eligible_for_funding: true,
+             course:,
+             lead_provider:,
+             cohort:)
+    end
+    let(:course) { create(:course, :sl) }
+    let(:lead_provider) { create(:lead_provider) }
+    let(:cohort) { create(:cohort, :current, :with_funding_cap) }
+
+    context "when application funded_place is false" do
```

Comment:
Can move updating the application in these specs into the before block? It reads oddly to me being "it sets the funded place to true" and we then change it to false right after this 😅 this order makes more sense in my head:

```
context "when the application funded_place is false" do
  before do
    params.merge!(funded_place: true)
    application.update!(funded_place: false)
  end

  it { expect { service.change }.to change { application.reload.funded_place }.to(true) }
end
```